src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldFilterTypes/SelectionFieldFilterType.js

```diff
@@ -123,10 +123,9 @@ class SelectionFieldFilterType extends AbstractFieldFilterType<?Array<string | n
                     />
                 }
                 {this.type === TYPE_SELECT &&
```

Comment:
@nnatter I think this is where calling it select starts feeling weird :see_no_evil: Referring to [your comment](https://github.com/sulu/sulu/pull/5174#issuecomment-608378649) in #5174.

Reply:
there is a `[single_select` type](https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SingleSelection.js#L66) in the `SingleSelection` too, so at least things would be consistent 🙂

Reply:
I don't see how this would be consistent. The `single_select` type is actually a  `SingleSelect` component, which is not the case here.

Reply:
From the user perspective:

There is a `single_select` field-type and a `single_selection` field-type. 
If i use a `single_selection` field-type with the `single_select` parameter, it looks like the `single_select` field-type.

Now there is a new `select` filter-type and a `selection` filter-type.
If I want to display a `selection` filter-type that looks like the `select` filter type, it would be consistent to use `select` as parameter value.

Reply:
The relationships between these two terms might be the same, but `select` still means something different in both cases, also from a user's perspective.

Reply:
I am not sure about that - to me, both of them are input elements that allow me to select options from a list of available options. What is the difference in your opinion?

Reply:
Keep in mind that the developer has to set this value, and if they ready `select` they probably gonna think of a HTML `select` element instead of a list of checkboxes. But I don't feel like this discussion leads anywhere, and I don't have a better idea anyway... As already stated in [the other PR](https://github.com/sulu/sulu/pull/5174) I guess I am going with `select` anyway.